### PR TITLE
FFWEB-1397 Create filter cloud demo

### PR DESCRIPTION
**Pull request by ghost**
The demo has been created, but webcomponents reference must be updated after version 3.8 release.

### CODE REVIEW

**Review by omikron-fp — CHANGES_REQUESTED** [review]
For a demo of the filter-cloud the filter-cloud it is quite small. I'm thinking of a more prominent presentation.
The example in the original issue should be a good inspiration.
https://github.com/FACT-Finder-Web-Components/ff-web-components/issues/21

**Comment by ghost** [comment]
> For a demo of the filter-cloud the filter-cloud it is quite small. I'm thinking of a more prominent presentation.
> The example in the original issue should be a good inspiration.
> [FACT-Finder-Web-Components/ff-web-components#21](https://github.com/FACT-Finder-Web-Components/ff-web-components/issues/21)

OK, I guess you mean only a size of the control, not the features it presents? Please, correct me if I'm wrong.

**Comment by omikron-fp** [comment]
I mean everything. Component area, font-size, padding, "x". And perhaps even a caption "Selected Filters".
So pretty much like in the screenshot.

**Comment by ghost** [comment]
> I mean everything. Component area, font-size, padding, "x". And perhaps even a caption "Selected Filters".
> So pretty much like in the screenshot.

Should be fine right now. Please verify.

File: ff-filter-cloud/styles.css [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+.fc-filter-item {
+    color: #5C5C5C;
+    background-color: #C7E4EC;
+    width: fit-content;
+    border-radius: 10px;
```
Comment (omikron-fp) [inline ff-filter-cloud/styles.css:216]: [optional] For consistency `5px` might fit better in the whole picture.

**Review by omikron-fp — APPROVED** [review]

File: ff-filter-cloud/styles.css (hunk 1 above)
Comment (ghost) [reply ff-filter-cloud/styles.css:216]: I will leave as it is now. With 5px radius they look like buttons.

**Pull request merged**